src/main/webapp/feed.js

```diff
@@ -176,88 +183,121 @@ function addEventInfo(event) {
 }
 
 function setChallengeBox(challengeId) {
```

Comment:
What IDE/Editor are you using?

Most of the changes in `setChallengesBox` `fillBadge` `createChallengeNavBarItem` `setChallengesNavBar` and `boldCurrentChallengeTitle` `findNextUncompletedChallenge`, etc.. seems to be re-indentation - perhaps we should reconfigure your editor to avoid re-indenting? It seems like most of the changes in this file is re-indentation, which makes it difficult for the reviewer to focus on what logic is actually being changed vs formatting changes.

One way to make reviews go more smoothly is to separate formatting PRs from PRs that contains logic changes. If you do intend to make formatting changes, put them in a separate PR solely containing formatting changes.

Reply:
That makes sense. I'm not sure how to separate the format changes into another PR for this round, But this something I will definitely keep in mind for the future.

Reply:
Please undo the formatting changes. You can do so by checking out the repository at an earlier state, copying the files to a temporary directory, checking out the `challenge_feature_2` branch, and pasting the old changes into the branch.